src/components/ConnectedList.jsx

```diff
@@ -87,7 +88,10 @@ class ConnectedList extends Component {
 
 const mapStateToProps = (state, ownProps) => {
   return {
-    connections: getConnections(state)
+    connections: sortBy(
+      getConnections(state),
+      ({ konnector }) => konnector.name
+    )
```

Comment:
Si ca vous pose vraiment un soucis d'inclure `lodash/sortBy` , je pourrais utiliser

```
const comparator = fn => (a, b) => fn(a) > fn(b) ? 1 : -1
getConnections(state).sort(comparator(({konnector}) => konnector.name))
```

Reply:
I am still against using lodash but I also want to be kind with developers who use it.
What @CPatchane think about it ?

Also, what do you think about sorting directly into `reducers/index.js` or better `ducks/connections/index.js` ? It encapsulate the whole process, we can avoid an explicit dependency into the component and we hide lodash into the duck.

Reply:
To me it is seldom the responsibility of the reducer to sort data, it is more the responsibility of the view. This is particularly true when the view lets you sort the data via buttons (sort by date, sort by most recent run) etc... This is why I put it there even if the view does not let you sort.

Reply:
I see your point, I think I should prefer having the sort mechanism into the duck and the sort option (or method) passed to the selector but let's go with it. Thanks @ptbrowne 👍